src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/UserManagementServicePermissionTests.cs

```diff
@@ -131,39 +131,15 @@ public class DeleteUserTests
             public async Task Success()
             {
                 await PolicyCheckBuilder<SecurityPolicies>()
+                    .SetupCheck(CanManageUsersOnSystem)
                     .AssertSuccess(async userService =>
                         {
                             var service = SetupUserManagementService(
-                                userService: userService.Object,
-                                enableDeletion: true);
+                                userService: userService.Object);
                             return await service.DeleteUser("ees-test.user@education.gov.uk");
                         }
                     );
             }
-
-            [Fact]
-            public async Task Forbidden_EnvironmentConfiguration()
-            {
-                await PolicyCheckBuilder<SecurityPolicies>()
-                    .AssertForbidden(async userService =>
-                        {
-                            var service = SetupUserManagementService(userService: userService.Object);
-                            return await service.DeleteUser("ees.test-user@education.gov.uk");
-                        }
-                    );
-            }
-
-            [Fact]
-            public async Task Forbidden_InvalidEmailAddressFormat()
-            {
-                await PolicyCheckBuilder<SecurityPolicies>()
-                    .AssertForbidden(async userService =>
-                        {
-                            var service = SetupUserManagementService(userService: userService.Object);
-                            return await service.DeleteUser("invalid-email-to-delete@education.gov.uk");
-                        }
-                    );
-            }
         }
 
         private static UserManagementService SetupUserManagementService(
```

Comment:
Now `appOptions` is gone, we need to remove it from here.

Reply:
weirdly didn't complain about this locally when running the tests, it's gone now.